src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolderDescriptor.java

```diff
@@ -41,6 +41,40 @@ public String getDisplayName() {
     }
 
     /**
+     * Needed if it wants Folders are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     * TODO: Replace to {@code NestedProjectsCategory.ID}
```

Comment:
AFAIK, `NestedProjectsCategory` will be provide by core though it can be provided by this plugin and additionally we can test the new API. Are you ok?